src/index.js

```diff
@@ -0,0 +1,265 @@
+  const selectedTasks = myTasks.filter((task) => task.projectsN === currentProject);
+
+  if (clicked.classList.contains('delProjectBtn')) {
+    alert('No kidding?!');
+    clicked.closest('article').remove();
+    deleteProject(myProjects, currentProject);
+    deleteAllTasks(currentProject);
+    setProjects();
+    setTasks();
+    makeToday();
+  }
+
+  if (clicked.classList.contains('editProjectBtn')) {
+    const project = myProjects.find((x) => x.name === currentProject);
+
+    getName.value = project.name;
+
+    getDescription.value = project.description;
+
+    editForm.addEventListener('submit', (e) => {
+      e.preventDefault();
+
+      const edited = new Project(editForm[0].value, editForm[1].value);
+
+      let confirmation = true;
+
+      if (edited.name === project.name && edited.description === project.description) {
+        alert("You haven't change anything");
+        confirmation = false;
+        return;
+      }
+
+      const checkData = edited.name;
+
+      if (checkProject(checkData) && edited.name !== project.name) {
+        alert('Project with same name already exists');
+
+        confirmation = false;
+      }
+
+      if (!confirmation) {
+        return;
+      }
+
+      project.name = edited.name;
+      project.description = edited.description;
+
+      setProjects();
+      projectsContainer.innerHTML = '';
+      projectsToDom();
+      addForm.reset();
+    });
+  }
+  if (clicked.classList.contains('seeBtn')) {
+    makeTaskSecOnClck(currentProject, selectedTasks);
+  }
+});
+
+first.addEventListener('click', (e) => {
+  e.preventDefault();
+
+  const clicked = e.target;
+  if (clicked.classList.contains('plus')) {
+    makeTaskForm();
+  }
+});
+
+const taskForm = document.getElementById('taskForm');
+taskForm.addEventListener('click', (e) => {
+  e.preventDefault();
+
+  const clicked = e.target;
+
+  if (clicked.classList.contains('dismissTask')) {
+    taskForm.innerHTML = '';
+    countForm = false;
+    return;
+  }
+
+  if (clicked.classList.contains('confirmTask')) {
+    const newTitle = taskForm[0].value;
+    const newComment = taskForm[1].value;
+    const newPriority = taskForm[2].value;
+    let newDate = taskForm[3].value;
+
+    let confirm = true;
+
+    if (!newTitle) {
+      alert('Title must not be empty');
+      confirm = false;
+      return;
+    }
+
+    if (confirm === false) {
+      // eslint-disable-next-line
+      return false;
+    }
+
+    if (!newDate) {
+      newDate = setDate();
+    }
+
+    const newTask = new Task(newTitle, currentProject, newComment, newPriority, newDate);
+
+    myTasks.push(newTask);
+    setTasks();
+    const selectedTasks = myTasks.filter((task) => task.projectsN === currentProject);
+
+    makeTaskSection(currentProject, selectedTasks);
+    makeToday();
+    taskForm.reset();
+  }
+});
+
+tableArticle.addEventListener('click', (e) => {
+  e.preventDefault();
+  const clicked = e.target;
+  // eslint-disable-next-line
```

Comment:
Kindly avoid the ESLInt inline disabling rule, it is the same as altering the ESLint config files